### ISSUE TITLE
Normalize is included in the middle of the CSS bundle

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#3565]
Since our style is fairly heavily specified I'm not sure how much this is a problem but normalize is not currently included at the top of our stylesheet.

**Comment by muffinresearch** [comment]
Looks like there's various issues reportd on webpack the text-extractor plugin related to this. It'll need further investigation but the problem probably lies within that or how we've got things configured.

**Comment by willdurand** [comment]
It looks like the order is inverted because the fonts, normalize and `App/styles.scss` are at the bottom of the stylesheet while `.Card-header` is at the top (it is not the first component rendered).

**Comment by willdurand** [comment]
ahm, I think I found the issue, more info at: https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/548.

TL;DR: we always `import './styles.scss'` after the JS imports, which means the SCSS/CSS imports in `App/index.js` are the last ones being imported. I believe it should always been imported after except for the `App` components.

**Comment by willdurand** [comment]
For QA: please make sure the website still uses the CSS style. If you like, you could look at the `.css` file source code and see the first lines starting with `@font-face ...`.

**Comment by vcarciu** [comment]
Verified as fixed : 
![css](https://user-images.githubusercontent.com/1583842/43520872-f2964e3e-959c-11e8-87a1-a5f2e123e05e.png)
